app/graphql/types/integrations/anrok.rb

```diff
@@ -27,6 +28,10 @@ def external_account_id
 
         object.api_key.split('/')[0]
       end
+
+      def failed_invoices_count
+        object.error_details.where(owner_type: 'Invoice').kept.count
```

Comment:
can you please also fix `models/integrations/anrok_integration`
`has_many :error_details, -> { where({error_details: {error_code: 'tax_error'}}) },` -> should be ->
`has_many :error_details, -> { where({details: {error_code: 'tax_error'}}) },`